src/Symfony/Component/Form/Extension/Core/Type/TelType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+
+class TelType extends AbstractType
```

Comment:
Put different; Symfony provides `TelephoneType` by implementing `type="tel"`. Point is; we should not name our class to an implementation detail :) IMHO.